code/game/objects/objs.dm

```diff
@@ -1,15 +1,15 @@
 /obj
 	//var/datum/module/mod		//not used
 	var/origin_tech = null	//Used by R&D to determine what research bonuses it grants.
-	var/crit_fail = 0
+	var/crit_fail = FALSE
 	animate_movement = 2
-	var/list/attack_verb = list() //Used in attackby() to say how something was attacked "[x] has been [z.attack_verb] by [y] with [z]"
+	var/list/attack_verb //Used in attackby() to say how something was attacked "[x] has been [z.attack_verb] by [y] with [z]"
```

Comment:
This is an `obj` level list that doesn't really have any reason to be.

Move it up to `/obj/item`

No one is going to be picking up a cargo console and bashing people with it (as hilarious as that sounds).

Reply:
QM's PISSED